### ISSUE TITLE
The input mask of unet is that before sigmoid

### ISSUE THREAD

**Issue by hexiaoyi95** [MCG-NJU/EMA-VFI#14]
https://github.com/MCG-NJU/EMA-VFI/blob/75b6f6a889e695df875e103374040d47a4cfac7c/model/flow_estimation.py#L137C65-L137C65
this is a bug or not?

**Comment by GuozhenZhang1999** [comment]
Thanks for the question. We think input features of unet stay on the same scale will help network learning, but not verify that. You can try to see the impact on performance.

**Comment by hexiaoyi95** [comment]
Thanks for your reply. Acctually I'm training your model with my own dataset from scratch. And find it's very hard to converge. Did you ever meet similar issues? Any experiences you can share?

**Comment by GuozhenZhang1999** [comment]
You can try to adjust batchsize and learning rate, and consider training with Vimeo if the amount of data is small.
